cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp

```diff
@@ -15,13 +15,17 @@
  */
 #pragma once
 
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+
 #include <librdkafka/rdkafkacpp.h>
 #include <algorithm>
 #include <chrono>
 #include <cudf/io/datasource.hpp>
 #include <map>
 #include <memory>
 #include <string>
+#include "kafka_callback.hpp"
```

Comment:
Will organize the imports. However isn't it recommended in the Python development headers to include the Python import as the first import? I can't include a direct link but in the python docs at [this](https://docs.python.org/3/c-api/intro.html) this is the section I was referring to. 
```
Note Since Python may define some pre-processor definitions which affect the standard headers on some systems, you must include Python.h before any standard headers are included.
It is recommended to always define PY_SSIZE_T_CLEAN before including Python.h. See Parsing arguments and building values for a description of this macro.
```

Reply:
OK, put that one first with a comment saying it must be first (so nobody moves it).